nixos/modules/security/pam.nix

```diff
@@ -219,6 +219,14 @@ let
         '';
       };
 
+      nodelay = mkOption {
+        default = false;
+	type = types.bool;
+	description = ''
+	  Wheather the delay after typing a wrong password should be disabled.
+	'';
```

Comment:
this indentation doesn't match

Reply:
Sorry, somehow a tab char got in there, is now fixed.